### DISCUSSION TITLE
Can I rejoin?

### DISCUSSION THREAD

**Discussion by zoraezimran** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#140] [Q&A] [answered]
I accidently left the org so can you add me back pls and give me write access

Also can you assign me for some of the issue things

**Comment by Shluffy** [comment, marked as answer]
no, but you can be an outside collaborator

**Reply by Shluffy** [reply]
and you left "by accident"?

**Reply by Shluffy** [reply]
no, but you can be an outside collaborator
